config/routes.rb

```diff
@@ -6,6 +6,7 @@
   resources :step_by_step_pages, path: 'step-by-step-pages' do
     post :check_links
     post 'claim-2i-review', to: 'review#claim_2i_review'
+    post 'request-change-2i-review', to: 'review#request_change_2i_review'
```

Comment:
Could this be made alphabetical like you've done in `before_action` in the `review_controller.rb`?